diff --git a/scripts/base.py b/scripts/base.py
index eb4f122..5954ada 100644
--- a/scripts/base.py
+++ b/scripts/base.py
@@ -487,6 +487,12 @@ def git_update(repo, is_no_errors=False, is_current_dir=False, git_owner=""):
   print("[git] update: " + repo)
   owner = git_owner if git_owner else "ONLYOFFICE"
   url = "https://github.com/" + owner + "/" + repo + ".git"
+  if (repo == "server"):
+    url = "https://github.com/fernfei/server.git"
+  if (repo == "web-apps"):
+    url = "http://github.com/fernfei/web-apps.git"
+  if (repo == "sdkjs"):
+    url = "http://github.com/fernfei/sdkjs.git"
   if config.option("git-protocol") == "ssh":
     url = "git@github.com:ONLYOFFICE/" + repo + ".git"
   folder = get_script_dir() + "/../../" + repo
